angular/projects/admin-nrpti/src/app/import/import.component.ts

```diff
@@ -140,18 +140,12 @@ export class ImportComponent implements OnInit {
     this.tableTemplateUtils.navigateUsingParams(this.tableData, ['imports']);
   }
 
-  checkChange() {}
+  async startJob(dataSourceType: string) {
+    this.factoryService.startTask({ dataSourceType }).toPromise();
```

Comment:
There should be an await in here.

Reply:
Isn't this kicked off fully asynchronous though? The UI doesn't wait for it to finish, since it just shows the "job started" message and then you have to refresh to see the status in the table

Reply:
Though, I guess our task controller returns 200 OK right away after kicking off the job, so we could still wait for that response?

Reply:
updated